index.html

```diff
@@ -2097,6 +2095,19 @@ <h4>Semantic Interoperability</h4>
 <code>@protected</code> feature in the JSON-LD 1.1 specification.
             </li>
           </ul>
+          <p>
+A human-readable document describing the expected order of values for the
+<code>@context</code> <a>property</a> SHOULD be available. A machine-readable
+description (i.e. a normal JSON-LD "context" document) SHOULD be available
+at the URL specified in the <code>@context</code> <a>property</a>.
+          </p>
+          <p>
+These requirements guarantee semantic interoperability between JSON and JSON-LD
+for terms defined by the <code>@context</code> mechanism. While JSON-LD processors
```

Comment:
I really like this last sentence and will be stealing it and integrating it into the PR I provided in an attempt to merge the best language in both PRs.

Reply:
Stolen in https://github.com/w3c/vc-data-model/pull/546/commits/0221fbb2eb2833505848649e65980e38110d0f55 :)